polaris/polaris/sep12/customer.py

```diff
@@ -1,40 +1,106 @@
+from typing import Dict
+
+from django.utils.translation import gettext as _
+from rest_framework.views import APIView
 from rest_framework.request import Request
 from rest_framework.response import Response
 from rest_framework.decorators import api_view, renderer_classes
 from rest_framework.renderers import JSONRenderer
 
-from polaris.utils import extract_sep9_fields, render_error_response
+from polaris.utils import extract_sep9_fields, render_error_response, Logger, memo_str
 from polaris.sep10.utils import validate_sep10_token
 from polaris.integrations import registered_customer_integration as rci
 
 
-@api_view(["PUT"])
-@renderer_classes([JSONRenderer])
-@validate_sep10_token("sep6")
-def put_customer(account: str, request: Request) -> Response:
-    if account != request.data.get("account"):
-        return render_error_response(
-            "The account specified does not match authorization token", status_code=403
-        )
-    try:
-        rci.put(
-            {
-                "account": request.data.get("account"),
-                "memo": request.data.get("memo"),
-                "memo_type": request.data.get("memo_type"),
-                **extract_sep9_fields(request.data),
-            }
-        )
-    except ValueError as e:
-        return render_error_response(str(e), status_code=400)
-    else:
-        return Response({}, status=202)
+logger = Logger(__name__)
+
+
+class CustomerAPIView(APIView):
+    renderer_classes = [JSONRenderer]
+
+    @staticmethod
+    @validate_sep10_token("sep6")
```

Comment:
Why are we referencing sep6 in a sep12 file?

Reply:
SEP-12 is used in SEP-6, and SEP-6 has all 403 permission denied responses to have this body:
```
{
    "type": "authentication_required"
}
```
So I added the `"sep6"` string to ensure thats the response body on authenticated requests. 

Since SEP-31 isn't opinionated about the response body of 403 responses it doesn't conflict.